src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -59,6 +59,7 @@ export class CalcitePickListItem {
   }
 
   /**
+   * @deprecated This will be removed in a future build. Replaced by textLabel.
```

Comment:
[Nitpick] We can probably remove this: `This will be removed in a future build. `